playbooks/deploy_daskhub.yml

```diff
@@ -0,0 +1,34 @@
+- hosts: localhost # Kubectl is configured from this machine
+  roles:
+    - deploy_hub
+    # - deploy_prometheus_stack
+  vars:
+    # Helm name of the service
+    hub_deployed_name: daskhub
+    # K8s namespace
+    hub_namespace: daskhub
+    # Helm chart version for Jhub (newer version may require newer kubernetes version >=1.17)
+    hub_version: "2022.8.1"
+    # config file for DaskHub helm chart (in "/roles/deploy_daskhub/files")
+    hub_config_file: ../deploy_daskhub/files/config.yaml
```

Comment:
So I would move aspects like `hub_config_file` `repo_name` `url` ....etc. that are DRY considerations rather than user changeable aspects to the very bottom of this file with maybe a comment to separate them

Since other considerations like Nvidia GPU driver version will change "frequently" in comparison